app/assets/javascripts/discourse/app/components/plugin-outlet.hbs

```diff
@@ -24,11 +24,15 @@
   </this.wrapperComponent>
 {{else if this.connectorsExist}}
   {{! The modern path: no wrapper element = no classic component }}
-  {{#each this.connectors as |c|}}
+  {{#each (this.getConnectors hasBlock=(has-block)) as |c|}}
```

Comment:
Where is `has-block` coming from?

Reply:
https://guides.emberjs.com/release/components/block-content/#toc_conditional-blocks